crates/polars-ops/src/series/ops/business.rs

```diff
@@ -7,14 +8,28 @@ use polars_core::prelude::*;
 /// - `start`: Series holding start dates.
 /// - `end`: Series holding end dates.
 /// - `week_mask`: A boolean array of length 7, where `true` indicates that the day is a business day.
+/// - `holidays`: timestamps that are holidays. Must be provided as i32, i.e. the number of
+///   days since the UNIX epoch.
 pub fn business_day_count(
     start: &Series,
     end: &Series,
     week_mask: [bool; 7],
+    holidays: &[i32],
 ) -> PolarsResult<Series> {
     if !week_mask.iter().any(|&x| x) {
         polars_bail!(ComputeError:"`week_mask` must have at least one business day");
     }
+
+    // De-dupe and sort holidays, and exclude non-business days.
```

Comment:
Why would we get duplicates?

Reply:
If a user passes them :) In which case, they'd get incorrect results if Polars doesn't de-dupe for them